README.md

```diff
@@ -20,8 +20,29 @@ Go and [register a new OAuth application](https://github.com/settings/applicatio
 
 Once you've created the application, you should be given the client id and the client secret. **Make a note of these**.
 
+Additionally, if the intention is to run the GitHub repository poller (`npm run poll-repositories`), a API token should be acquired so the API calls are submitted to an acceptable rate-limit (5k req/s instead of 60 req/s for anonymous access)::
+
+  curl -s -u '<GH_USER>' https://api.github.com/authorizations \
+    -H 'Content-Type: application/json' \
+    -H 'X-GitHub-OTP: <GH 2FA CODE IF ENABLED>' \
+    -d '{"client_id": "c54ee622a3fa9622701f", \
+         "client_secret": "2a6010d06c7c90b1fcf6518d72eac0bab64cf35b", \
+	 "note": "Build.s.io poller token"}' | jq '.token' -r
+
+`curl` will ask for you GitHub password (basic-auth) and acquire a 40-char authorization token. Make note of it, it should be set in the running environment as `GITHUB_AUTH_CLIENT_TOKEN`.
```

Comment:
Current `GITHUB_AUTH_CLIENT_TOKEN` is optional, it will give us enough flexibility to move forward. I presume we do want the safety of isolated rate-limits for the webhook processor and the poller.